db/seeds.rb

```diff
@@ -42,8 +42,7 @@
                                              .merge(user: supervisor)
       supervisor_report = create :incident_report, supervisor_report_attrs
       incident_attrs[:supervisor_incident_report] = supervisor_report
-      incident_attrs[:supervisor_report] = create :supervisor_report,
-                                                  user: supervisor
+      incident_attrs[:supervisor_report] = create :supervisor_report
```

Comment:
Why isn't a supervisor connected to a supervisor report? That's what I understand the change to mean. 

Edit: Got it